test/test_app.rb

```diff
@@ -344,6 +344,45 @@
     Precious::App.set(:wiki_options, { :template_page => false })
   end
 
+  test "create with template filter without parameter succeed" do
```

Comment:
Thanks for including the tests! You can omit the `succeed` in the test names :)

Reply:
Sure, that's cleaner.